src/index.ts

```diff
@@ -296,15 +352,23 @@ class FileBrowser extends Widget {
       return;
     }
```

Comment:
rather than assume breadcrumbs is the first child, you can query by classname which is more robust in the face of change.
`this.node.getElementsByClassName(BREADCRUMBS_CLASS)[0]`